wp-content/themes/core/components/accordion/accordion.php

```diff
@@ -0,0 +1,55 @@
+<?php
+// phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
+$controller = \Tribe\Project\Templates\Components\accordion\Controller::factory( $args );
+?>
+<div
+	<?php echo $controller->container_classes(); ?>
+		role="tablist"
+		aria-multiselectable="true"
+	<?php echo $controller->container_attrs(); ?>
+>
+	<?php foreach ( $controller->rows as $key => $row ) { ?>
```

Comment:
This is super minor, but I prefer the alternate loop syntax where ever we are using php open & close tags within the loop: `<?php foreach ( $controller->rows as $key => $row ): ?> ... <?php endforeach; ?>`. Curious to hear other's thoughts on this.

Reply:
I disagree. Largely because in the IDE it's easy to put a cursor on `}` and have it highlight the matching `{`.

Reply:
I also prefer `foreach` 😂

Reply:
I prefer `{  }` as well, 1 char, loop block highlighting, and matches js/other langs